test/calibration/test_setup_library.py

```diff
@@ -75,3 +72,51 @@ def test_turn_off_drag(self):
 
         library = FixedFrequencyTransmon()
         self.assertTrue(isinstance(library["x"].blocks[0].pulse, pulse.Drag))
+
+    def test_unlinked_parameters(self):
+        """Test the we get schedules with unlinked parameters."""
+
+        library = FixedFrequencyTransmon(link_parameters=False)
+
+        sched_x = library["x"]
+        sched_y = library["y"]
+        sched_sx = library["sx"]
+        sched_sy = library["sy"]
+
+        # Test the number of parameters.
+        self.assertEqual(len(sched_x.parameters & sched_y.parameters), 2)
+        self.assertEqual(len(sched_sx.parameters & sched_sy.parameters), 2)
+
+        expected = [
+            (0.5, "amp", (), "x"),
+            (0.0, "β", (), "x"),
+            (160, "duration", (), "x"),
+            (40, "σ", (), "x"),
+            (160, "duration", (), "sx"),
+            (0.0, "β", (), "sx"),
+            (0.25, "amp", (), "sx"),
+            (40, "σ", (), "sx"),
+            (0.5j, "amp", (), "y"),
+            (0.0, "β", (), "y"),
+            (160, "duration", (), "y"),
+            (40, "σ", (), "y"),
+            (160, "duration", (), "sy"),
+            (0.0, "β", (), "sy"),
+            (0.25j, "amp", (), "sy"),
+            (40, "σ", (), "sy"),
+        ]
+
+        self.assertSetEqual(set(library.default_values()), set(expected))
+
+    def setup_partial_gates(self):
```

Comment:
This method needs to start with `test` in order to be run as a test.

Reply:
Good catch. Thanks. See here: a457f1a